build/helpers/file-includes-all-subdirectories-as-named-exports.ts

```diff
@@ -0,0 +1,33 @@
+import * as path from "path";
+import * as fs from "fs";
+import { pascalCase } from "../../packages/fast-web-utilities";
+
+/**
+ * Verifies all components in source directory are exported
+ */
+export function includesAllSubdirectoriesAsNamedExports(indexFile: string): boolean | Error {
```

Comment:
You're not really returning an error here - you're throwing an error. This should just be boolean.

Reply:
Got it.